shared/js/actions.js

```diff
@@ -145,6 +145,15 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Notifies that a tile has been received from the other peer.
+     */
+    ReceivedAddPage: Action.define("receivedAddPage", {
```

Comment:
Just to verify, you want this to look like:

    AddedPage: Action.define("addedPage", {

Right?